Core/Utilities/test/Test_Encode.py

```diff
@@ -0,0 +1,160 @@
+"""Test Encoding function of DIRAC
+Any library that would pretend replacing diset should pass this tests.
+
+"""
+
+
+from string import printable
+import sys
+
+from DIRAC.Core.Utilities.DEncode import encode as disetEncode, decode as disetDecode, g_dEncodeFunctions
+
+from hypothesis import given
+from hypothesis.strategies import integers, lists, recursive, floats, text,\
+    booleans, none, dictionaries, tuples, datetimes
+
+from pytest import mark, approx
```

Comment:
Can't you simply use unittest?

Reply:
pytest supports all unittest feature and tests (so no need t o migrate existing tests), but it is so much more powerful. And since we run all our tests with it, I might as well use it.